src/env.js

```diff
@@ -261,7 +261,12 @@ Sk.output = function (x) {
  * todo; this should be an async api
  */
 Sk.read = function (x) {
-    throw "Sk.read has not been implemented";
+    if (Sk.builtinFiles === undefined) {
+        throw Sk.builtin.NotImplementedError("Sk.read has not been implemented");
```

Comment:
should we say that `-stdlib.js` wasn't loaded? instead of saying it's not implemented?

Reply:
Any preference on the error type?

Reply:
I don’t have a preference. 😁